rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/WorkerSourceTask.java

```diff
@@ -351,7 +352,11 @@ private void putExtendMsgProperty(ConnectRecord sourceDataEntry, Message sourceM
             return;
         }
         for (String key : keySet) {
-            MessageAccessor.putProperty(sourceMessage, "connect-ext-" + key, extensionKeyValues.getString(key));
+            if (key.equals(MessageConst.PROPERTY_KEYS) || key.equals(MessageConst.PROPERTY_TAGS)) {
```

Comment:
imo, we should have a static list of properties and see if the key is included in this list instead of enumerating.
For example: final static whiteListKey = {tags, propertyKey}
whiteListKey.cotain(key)

Reply:
Well, that's more flexible，I will change it later.